clients/hadoopfs/src/test/java/io/lakefs/FSTestBase.java

```diff
@@ -60,7 +60,7 @@
  * See e.g. "Base Test Class Testing Pattern: Why and How to use",
  * https://eliasnogueira.com/base-test-class-testing-pattern-why-and-how-to-use/
  */
-abstract class FSTestBase {
+public abstract class FSTestBase {
```

Comment:
Please document where this class is being used. This is easily revertible and breaks the usage you have in mind here